src/dispatch/static/dispatch/src/case/EditSheet.vue

```diff
@@ -11,6 +11,10 @@
               Reported - {{ reported_at | formatRelativeDate }}
             </v-list-item-subtitle>
           </v-list-item-content>
+          <v-btn @click="showEscalateDialog(selected)" color="error">
+            <v-icon left> error_outline </v-icon>
+            Escalate Case as Incident
```

Comment:
Incident is the only thing that a case can be escalated to, so I'd suggest we remove it
```suggestion
            Escalate Case
```

Reply:
It also helps with having a smaller button.